packages/react-frontend/src/index.css

```diff
@@ -56,3 +56,83 @@ h2 {
   font-size: 240px; 
   font-weight: bold;
 }
+
+footer {
+  background-color: #690606;
+  padding-top: 6px; /* Only top padding */
+  width: 100vw;
+  box-sizing: content-box; /* Ensures padding is included in the width */
```

Comment:
i think using 'border-box' could be better than 'content-box' for box-sizing, but let me know what u think